app/src/main/java/edu/uc/zhukv/droneradarmap/ui/home/NotificationsFragment.java

```diff
@@ -0,0 +1,4 @@
+package edu.uc.zhukv.droneradarmap.ui.home;
+
+public class NotificationsFragment {
```

Comment:
Created a new Class due to the fragment_notifications.xml. The "tools:context" was not in use, which caused an error.

Reply:
This is an empty class, which is incomplete functionality.
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.